examples/kustomize.jsonnet

```diff
@@ -1,5 +1,6 @@
 local kp =
-  (import 'kube-prometheus/kube-prometheus.libsonnet') + {
+  (import 'kube-prometheus/kube-prometheus.libsonnet') + 
+  (import 'kube-prometheus/kube-prometheus-aws-eks-cni.libsonnet') + {
```

Comment:
this example should be generic and not specific to eks

Reply:
d32e859a11b154bfdd29c8341a0d39cdab499215 revert back to the generic example.